### PR TITLE
TravisCI integration passes...

### LINKED ISSUE #5: not Travis / CI
Would be wonderful to have a `CI` going; imagine `selenium` changing the traits and subsequently validating that a linked `trait` was updated shortly after the invoked change. 

Being able to invoke a `CI` build weekly, will also implicitly report the integrity of the tools, frameworks and utilities involved.

### PR DESCRIPTION
**Pull request by jf---**
trivial selenium test is the basis for JS test integration
closes #5 #3

### CODE REVIEW

**Comment by jf---** [comment]
Next stop; build run the `python setup.py build_py` command such that building the JS dependencies is invoked via `npm` || `yarn`... 
That said, I'm feeling a little clueless on #4

**Comment by jf---** [comment]
Hi @maartenbreddels any thoughts on the PR?

**Comment by maartenbreddels** [comment]
I pushed some changes, fixing #4 I hope, would be great if you could rebase!

**Comment by jf---** [comment]
thx for the swift response, stoked to see progress on #4...
I've rebased, though I'd like to plough on and get #7 and #8 out of the way before merging, seems like the sensible thing to do

File: flask_ipywidgets/example2.py [hunk 1]
```diff
@@ -22,7 +22,4 @@ def index():
 
 
 if __name__ == "__main__":
-    from gevent import pywsgi
-    from geventwebsocket.handler import WebSocketHandler
-    server = pywsgi.WSGIServer(('', 5000), app, handler_class=WebSocketHandler)
-    server.serve_forever()
+    flask_ipywidgets.main_factory(app)()
```
Comment (rs2) [inline flask_ipywidgets/example2.py]: Trailing newlines are not strictly required by [PEP-8](https://www.python.org/dev/peps/pep-0008/#blank-lines), but are recommended.

Ditto re `.travis.yaml` above.

File: flask_ipywidgets/example3.py [hunk 2]
```diff
@@ -50,8 +51,4 @@ def index():
 
 
 if __name__ == "__main__":
-    from gevent import pywsgi
-    from geventwebsocket.handler import WebSocketHandler
-
-    server = pywsgi.WSGIServer(('', 5000), app, handler_class=WebSocketHandler)
-    server.serve_forever()
+    flask_ipywidgets.main_factory(app)()
```
Comment (rs2) [inline flask_ipywidgets/example3.py]: Ditto.

File: flask_ipywidgets/example2.py (hunk 1 above)
Comment (jf---) [reply flask_ipywidgets/example2.py]: Thx @rs2 will follow up shortly.

**Comment by jf---** [comment]
superseded by #10

**Pull request closed without merging**